project/MiMaSettings.scala

```diff
@@ -0,0 +1,18 @@
+package sbtbuild
+
+import sbt._
+import com.typesafe.tools.mima.core._
+import com.typesafe.tools.mima.plugin.MimaPlugin.autoImport._
+
+object MimaSettings {
+  val mimaSettings = Def.settings (
+    mimaBinaryIssueFilters ++= Seq(
+      ProblemFilters.exclude[IncompatibleResultTypeProblem]("sbt.TupleSyntax.t*ToApp*"),
+      ProblemFilters.exclude[IncompatibleResultTypeProblem]("sbt.TupleSyntax.t*ToTable*"),
+      ProblemFilters.exclude[MissingClassProblem]("sbt.Scoped$Apply*"),
```

Comment:
Don't we need to care about these?

Reply:
If I did it I'm 90% sure I did it because it's safe.

But with the distance of time I couldn't tell you what that reason is.  I'll go back and find out.

Reply:
I can't quite tell if removing these would be unsafe, and I also can't find a way to change this PR so it doesn't introduce any bincompat changes.  So I'll just leave it be.